docs/rule/use-concat-for-string-concatenation.md

```diff
@@ -0,0 +1,24 @@
+# use-concat-for-string-concatenation
+
+This rule requires the usage of `concat` helper to build strings for element attributes, which ideally prevents common formatting errors (i.e. CMD + P + `Format`) with plain concatenation.
+
+## Forbidden
+
+```hbs
+<div class="flex-{{if this.show "100" "50"}}"></div>
```

Comment:
thanks for the PR @betocantu93. 

I have to strongly disagree with this rule though... 😅 

I think it might be better to fix the automatic formatter that you're using instead of making the templates more complex because of the formatter :)

Reply:
that makes sense, thanks @Turbo87 for the feedback, it works for us for now, until we fix the formatter